Getting_Started_with_eva_sub_cli.md

```diff
@@ -0,0 +1,127 @@
+
+File format version
+
+```
+##fileformat=VCFv4.2
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Total Depth">
+##FILTER=<ID=PASS,Description="All filters passed">
+##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
+
+```
+
+Body: The body of the VCF file contains the actual variant data, with each row representing a single variant. The columns in the body are : CHROM, POS, ID, REF, ALT, QUAL, FILTER, INFO, FORMAT, Sample Columns
+
+```
+#CHROM  POS  ID  REF  ALT  QUAL  FILTER  INFO  FORMAT  [SampleIDs...]
+``` 
+Here's a small example to illustrate the structure of a VCF file: Example VCF file archived at EVA to be inserted 
+
+# Metadata Spreadsheet
+
+The spreadsheet provides comprehensive contextual information about the dataset, ensuring that each submission is accompanied by detailed descriptions that facilitate proper understanding and use of the data. Key elements included in the metadata spreadsheet are analysis and project information, sample information, sequencing methodologies, experimental details. 
+
+
+| WORKSHEET           | EXPLANATION          |
+| -----------------   | -------------------- |
+|  Submitter Details   | This sheet captures the details of the submitter|
+| Project           | The objective of this sheet is to gather general information about the Project including submitter, submitting centre, collaborators, project title, description and publications. |
+| Sample            | Projects consist of analyses that are run on samples. We accept sample information in the form of BioSample, ENA or EGA accession(s). We also accept BioSamples sampleset accessions. If your samples are not yet accessioned, and are therefore novel, please use the "Novel sample(s)" sections of the Sample(s) worksheet to have them registered at BioSample |
+| Analysis          | For EVA, each analysis is one vcf file, plus an unlimited number of ancillary files. This sheet allows EVA to link vcf files to a project and to other EVA analyses. Additionally, this worksheet contains experimental meta-data detailing the methodology of each analysis. Important to note; one project can have multiple associated analyses          |
+| Files             | Filenames and associated checking data associated with this EVA submission should be entered into this worksheet. Each file should be linked to one, or more, analysis.            |
+
+
+# Validation checks 
+
+The CLI tool performs the following validation checks and generates corresponding reports:
+
+- Metadata check to ensure that the metadata fields have been correctly filled in
+- VCF check to ensure that the VCF file follows the VCF format specification
+- Assembly check to ensure that the genome and the VCF match
+- Sample name check to ensure that the samples in the metadata can be associated with the sames in the VCF
+
+In the following sections, we will examine each of these checks in detail, starting with the Metadata check.
+
+# Metadata check
```

Comment:
```suggestion
## Metadata check
```
Optional, this makes a smaller header, so this becomes a subsection of "Validation checks"... would do this for all the other three headers as well.